4.3/contrib/docker-compose-planet.yml

```diff
@@ -0,0 +1,32 @@
+version: "3"
+
+# For a full planet instance, we apply some best practices from the documentation (like using a flatnode file).
+# This compose file uses bind mounts, so it'll reference /data. You can either also mount the storage (for DB + flat node file) under /data or change it.
+# Minimum specification for a machine running the import (if this succeeded on a lower spec machine, please contribute them):
+# - 16 core CPU (set THREADS variable to number of cores/threads available)
+# - 64GB RAM
+# - 1.5TB (NVMe) SSD storage
+
+services:
+    nominatim:
+        container_name: nominatim
+        image: mediagis/nominatim:4.3
+        restart: always
```

Comment:
can you remove the restart policy, since we had problems in the past with people not checking their import process when it fails? thanks

Reply:
sure thing, I removed it now